system/jlib/jptree.cpp

```diff
@@ -4528,6 +4528,17 @@ IPropertyTree *createPTree(IBufferedSerialInputStream &src, byte flags)
     return tree;
 }
 
+IPropertyTree *createPTree(IBufferedSerialInputStream &src, const char *tag, IPTreeNodeCreator *nodeCreator)
```

Comment:
you should not need to know the tag of the create being deserialized..
It is in the stream.